ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -50,6 +47,16 @@ interface {{ vlan_interface.name }}
 {%             endfor %}
 {%         endif %}
 {%     endif %}
+{%     if vlan_interface.ip_verify is arista.avd.defined %}
+{%         if vlan_interface.ip_verify.unicast_source is arista.avd.defined %}
+{%             if vlan_interface.ip_verify.unicast_source.reachable_via is arista.avd.defined %}
+   ip verify unicast source reachable-via {{ vlan_interface.ip_verify.unicast_source.reachable_via }}
+{%             endif %}
+{%         endif %}
+{%     endif %}
+{%     if vlan_interface.ip_directed_broadcast is arista.avd.defined(true) %}
```

Comment:
unrelated change?

Reply:
its for the sequence, I checked on EOS

Reply:
We should not be changing ordering of existing commands. Just insert the new command the best you can and ignore the existing.

Reply:
reverted it